packages/edit-navigation/src/components/header/index.js

```diff
@@ -2,40 +2,97 @@
  * WordPress dependencies
  */
 import { __ } from '@wordpress/i18n';
-import { Button, Dropdown } from '@wordpress/components';
+import {
+	Spinner,
+	Button,
+	SelectControl,
+	Dropdown,
+} from '@wordpress/components';
 
 /**
  * Internal dependencies
  */
 import ManageLocations from './manage-locations';
+import AddMenuForm from './add-menu-form';
 
-export default function Header( { onBeginAddingMenu } ) {
+export default function Header( {
+	menus,
+	selectedMenuId,
+	onSelectMenu,
+	onCancelAddingMenu,
+} ) {
 	return (
 		<div className="edit-navigation-header">
 			<h1>{ __( 'Navigation' ) }</h1>
 
-			<Button
-				className="edit-navigation-header__add-new-button"
-				isSecondary
-				onClick={ onBeginAddingMenu }
-			>
-				{ __( 'Add new' ) }
-			</Button>
-
-			<Dropdown
-				contentClassName="edit-navigation-header__manage-locations"
-				position="bottom center"
-				renderToggle={ ( { isOpen, onToggle } ) => (
-					<Button
-						isLink
-						aria-expanded={ isOpen }
-						onClick={ onToggle }
-					>
-						{ __( 'Manage locations' ) }
-					</Button>
+			<div className="edit-navigation-header__actions">
+				{ menus ? (
+					<div className="edit-navigation-header__current-menu">
+						<p>{ __( 'Currently editing' ) }</p>
```

Comment:
`SelectControl` will render this for us if we remove the `hideLabelFromVision` prop.

Reply:
Good to know. I had to use a p tag to avoid some listing. I'd expect a label.

Reply:
c19c4498d0